src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/HotReload/IProjectBuildManager.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.HotReload;
+
+/// <summary>
+/// Provides functionality to build a project during hot reload operations.
+/// </summary>
+/// <remarks>
+/// This interface is used by the Hot Reload system to build projects before applying
+/// changes or restarting applications. It abstracts the underlying build mechanism
+/// and allows for proper integration with Visual Studio's build system.
+/// </remarks>
+[ProjectSystemContract(ProjectSystemContractScope.ProjectService, ProjectSystemContractProvider.System, Cardinality = ImportCardinality.ExactlyOne)]
+internal interface IProjectBuildManager
```

Comment:
Why rename it from IProjectHotReloadBuildManager if it's for HotReload purposes? This is not a general build manager interface, is it?